queries/python/highlights.scm

```diff
@@ -46,14 +46,28 @@
               "UserWarning" "DeprecationWarning" "PendingDeprecationWarning" "SyntaxWarning" "RuntimeWarning"
               "FutureWarning" "ImportWarning" "UnicodeWarning" "BytesWarning" "ResourceWarning"))
 
-; Function calls
-
-(decorator) @function
-((decorator (attribute (identifier) @function))
- (#match? @function "^([A-Z])@!.*$"))
-(decorator) @function
-((decorator (identifier) @function)
- (#match? @function "^([A-Z])@!.*$"))
+;; Decorators
+
+; Ex: @cache
+(decorator
+  (identifier) @annotation)
+
+; Ex: @functools.cache
+((decorator
+   (attribute) @annotation)
+ (#set! "priority" 105))
+
+; Ex: @lru_cache(maxsize=128)
+((decorator
+  (call function: (identifier) @annotation))
+ (#set! "priority" 105))
```

Comment:
**n.b.** This `priority` can be lifted if we move the rule to the end of this file. I wasn't able to lift custom priority for the other 2 rules. :(